presto-spi/src/main/java/com/facebook/presto/spi/function/RoutineCharacteristics.java

```diff
@@ -68,6 +80,16 @@ public boolean isCalledOnNullInput()
         return nullCallClause == CALLED_ON_NULL_INPUT;
     }
 
+    public Builder toBuilder()
```

Comment:
This is a little strange. It sounds like converting an object to a builder, where you actually mean creating a builder from this object. I prefer `builder().from(..)` or just `builder(...)`.

Reply:
Done